src/include/catalog/node_table_schema.h

```diff
@@ -25,6 +25,8 @@ class NodeTableSchema final : public TableSchema {
     inline const common::table_id_set_t& getFwdRelTableIDSet() const { return fwdRelTableIDSet; }
     inline const common::table_id_set_t& getBwdRelTableIDSet() const { return bwdRelTableIDSet; }
 
+    std::string ToCypher() const;
```

Comment:
NodeTableSchema and relTableSchema has a base class TableSchema, you can make it a base class method.

Reply:
relTableSchema has function parameters `std::string toCypher(std::string srcTableName, std::string dstTableName) const`; while NodeTableSchema does not have  `std::string toCypher() const;` What is more, there exists other classes inherit TableSchema, such as rel_table_gropu_schema and rdfgraphschema. I think it is unnecessary to make it a base method